_posts/2020/2020-11-26-how-load-testing-allows-to-measure-the-impact-of-code-changes.md

```diff
@@ -89,15 +90,15 @@ Check out the GitHub repo to see a list of available options, it is a very flexi
 
 ### Artillery
 
-The last tool and the most complete is called [Artillery](https://artillery.io/){:target="_blank"}.
-It can be found here: [Artillery Github Repo](https://github.com/artilleryio/artillery){:target="_blank"}
+The second tool and the most complete one is called [Artillery](https://artillery.io/){:target="_blank"}.
+It can be found here: [Artillery GitHub Repo](https://github.com/artilleryio/artillery){:target="_blank"}.
 
-It is available as a NPM (Node Package Manager) package, so you must either have it installed or run it from inside a docker container (believe me, this is possible, and can save time and space in your disk).
-The *npm* command to install it is as follows: `$ npm install -g artillery`
+It is available as an NPM (Node Package Manager) package, so you must either have it installed or run it from inside a docker container (believe me, this is possible, and can save time and space in your disk).
```

Comment:
It really should be "a NPM package" instead of "an NPM package".
The rule is that "a" becomes "an" when the following word starts with a vowel. So: "I keep **an** eye on it", but "he took **a** deep breath".

Reply:
Clarified in person :)